.github/workflows/docker-x86.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and test x86 Docker Images
```

Comment:
Regarding referencing this as x86, when using docker buildx, we refer to x86-64 as amd64 I think. I suggest we stick with the amd64 reference for consistency.

Reply:
Done